### PR TITLE
Pubkey resource unscoped by account

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Rename pubkey reservation dao methods.

This might be bit too much to add prefix, suffix might have enough visibility, happy to change to whatever we agree on :)

### CODE REVIEW

File: internal/dao/tests/pubkey_resource_test.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func createPubkeyResourceAzure() *models.PubkeyResource {
 	}
 }
 
-func setupResource(t *testing.T) (dao.PubkeyResourceDao, context.Context) {
+func setupPubkeyResource(t *testing.T) (dao.PubkeyResourceDao, context.Context) {
```
Comment (ezr-ondrej) [inline internal/dao/tests/pubkey_resource_test.go:36]: I've also renamed this, as when I searched for usage, this got me confused and I thought I'm renaming wrong method.
Hope it's ok :)

File: internal/dao/dao_interfaces.go [hunk 2]
```diff
@@ -42,12 +42,15 @@ type PubkeyDao interface {
...
 type PubkeyResourceDao interface {
-	GetResourceByProviderType(ctx context.Context, pubkeyId int64, provider models.ProviderType) (*models.PubkeyResource, error)
-	ListByPubkeyId(ctx context.Context, pkId int64) ([]*models.PubkeyResource, error)
-	Create(ctx context.Context, pkr *models.PubkeyResource) error
-	Delete(ctx context.Context, id int64) error
+	UnscopedGetResourceByProviderType(ctx context.Context, pubkeyId int64, provider models.ProviderType) (*models.PubkeyResource, error)
+	UnscopedListByPubkeyId(ctx context.Context, pkId int64) ([]*models.PubkeyResource, error)
+	UnscopedCreate(ctx context.Context, pkr *models.PubkeyResource) error
+	UnscopedDelete(ctx context.Context, id int64) error
 }
```
Comment (lzap) [inline internal/dao/dao_interfaces.go:54]: Wow that's quite harsh, I like it! No one will ever miss the fact that a DAO function is not scoped.

**Review by lzap — APPROVED** [review]
I thought a suffix is little bit less intrusive but I LOVE THIS.

File: internal/dao/dao_interfaces.go (hunk 2 above)
Comment (ezr-ondrej) [reply internal/dao/dao_interfaces.go:54]: @adiabramovitch opinions on this? I'm fine suffix or prefix, I just wanted to go with more strict to provoke a though and see what y'all think :)

**Pull request merged**